src/components/ErrorBoundary.tsx

```diff
@@ -25,10 +30,10 @@ class ErrorBoundary extends Component<any, any> {
 
   public static getDerivedStateFromError(_: Error): any {
     // Update state so the next render will show the fallback UI.
-    if (window.location.hostname !== 'paret0') {
-      return { hasError: true, errorText: _.message, stackTrace: '' };
+    if (window.location.hostname !== "paret0") {
```

Comment:
Refer to my comment number 1 in this PR on what to do about this component. Thank you for not deleting the class component outright, shows great prudence there.

Reply:
Ok, deleting something like this, just didn't seem like the right thing to do.

Reply:
@mikhael28 Im really struggling on this honestly, I can make errorboundary "work" but I dont think its working. useNavigate that replaced useHistory is a function and not an object now.  

This is what I have for location, navigate, and well params but that doesnt matter here. 
![image](https://user-images.githubusercontent.com/63400531/192716147-823992d4-3a62-4726-8d0c-66033a913cd7.png)

Reply:
Ok, I THINK I figured it out, I had to make a custom router and some other stuff. I dont have any errors any more and everything loads anyways.